client/src/components/Shared/Spinner/spinner.style.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export default styled.div`
```

Comment:
It's good practice to export as const instead of default as usually there are a lot of components. so `export const Spinner = styled.div` etc. And then change to a named import in index.

Reply:
I just changed it to default because eslinter keeps asking to change const to default if its only one element